### ISSUE TITLE
On the table card menu, add an entry to open the Table Permissions modal

### ISSUE THREAD

**Issue by seancolsen** [mathesar-foundation/mathesar#4070]
The Table Permissions modal is currently triggered via this button:

![image](https://github.com/user-attachments/assets/8d51615d-3285-4431-b70d-eb82562c3314)

During user testing, I noticed that users might also expect to find it here:

![image](https://github.com/user-attachments/assets/7d160c75-9611-4047-848e-bec0803aede5)

We should add a menu entry there which opens the Table Permissions modal.

**Comment by mathemancer** [comment]
Yes please! I think I try that menu every single time I try to edit table permissions.

**Comment by Sharath-1517** [comment]
Hey @seancolsen I'd like to work on this issue.

**Comment by vishaljoshi789** [comment]
Hi @seancolsen,
Here TablePermission is a component which takes table from the context, but when opening from menu there is no table in the context so,
Should I save the table on the context or,
Should I change the TablePermission component to take table as a prop.

**Comment by seancolsen** [comment]
@vishaljoshi789 take table as a prop.

In `TablePermissions.svelte` I see:

```ts
const tabularData = getTabularDataStoreFromContext();

$: table = $tabularData.table;
```

We ought to be able to change that to 

```ts
export let table: Table;
```

Then put a `TablePermissions` component inside each `TableCard`.
